inst/@sym/ezplot.m

```diff
@@ -126,58 +135,66 @@
 end
 
 
+%!shared hf
+%! hf = figure ('visible', 'off');
```

Comment:
This will probably be perfectly fine for some users, and cause catastrophic crashes for others (any Windows user, Nvidia users on Linux).